azkaban-common/src/main/java/azkaban/user/UserUtils.java

```diff
@@ -22,4 +33,56 @@ public static boolean hasPermissionforAction(final UserManager userManager, fina
 
     return false;
   }
+
+  /**
+   * Creates a watch thread which listens to specified files' modification and reloads
+   * configurations
+   */
+  public static void setupWatch(final String fileName, Logger log, ParseConfigFile parser) {
```

Comment:
should we expose log as an argument given this method will be exposed publicly? Consider cases when caller is using slf4j as its logging framework.

Reply:
I will add logger in Utils.